docs/app/Examples/elements/Flag/Types/FlagTableExample.js

```diff
@@ -1,1236 +1,1236 @@
+    alias: '',
   },
   {
-    "name": "tunisia",
-    "country_Code": "tn",
-    "alias": ""
+    name: 'tunisia',
+    country_code: 'tn',
+    alias: '',
   },
   {
-    "name": "turkey",
-    "country_Code": "tr",
-    "alias": ""
+    name: 'turkey',
+    country_code: 'tr',
+    alias: '',
   },
   {
-    "name": "turkmenistan",
-    "country_Code": "tm",
-    "alias": ""
+    name: 'turkmenistan',
+    country_code: 'tm',
+    alias: '',
   },
   {
-    "name": "tuvalu",
-    "country_Code": "tv",
-    "alias": ""
+    name: 'tuvalu',
+    country_code: 'tv',
+    alias: '',
   },
   {
-    "name": "uae",
-    "country_Code": "ae",
-    "alias": "united arab emirates"
+    name: 'uae',
+    country_code: 'ae',
+    alias: 'united arab emirates',
   },
   {
-    "name": "uganda",
-    "country_Code": "ug",
-    "alias": ""
+    name: 'uganda',
+    country_code: 'ug',
+    alias: '',
   },
   {
-    "name": "ukraine",
-    "country_Code": "ua",
-    "alias": ""
+    name: 'ukraine',
+    country_code: 'ua',
+    alias: '',
   },
   {
-    "name": "united.kingdom",
-    "country_Code": "gb",
-    "alias": ""
+    name: 'united.kingdom',
+    country_code: 'gb',
+    alias: '',
   },
   {
-    "name": "united.states",
-    "country_Code": "us",
-    "alias": "america"
+    name: 'united.states',
+    country_code: 'us',
+    alias: 'america',
   },
   {
-    "name": "uruguay",
-    "country_Code": "uy",
-    "alias": ""
+    name: 'uruguay',
+    country_code: 'uy',
+    alias: '',
   },
   {
-    "name": "us.minor.islands",
-    "country_Code": "um",
-    "alias": ""
+    name: 'us.minor.islands',
+    country_code: 'um',
+    alias: '',
   },
   {
-    "name": "us.virgin.islands",
-    "country_Code": "vi",
-    "alias": ""
+    name: 'us.virgin.islands',
+    country_code: 'vi',
+    alias: '',
   },
   {
-    "name": "uzbekistan",
-    "country_Code": "uz",
-    "alias": ""
+    name: 'uzbekistan',
+    country_code: 'uz',
+    alias: '',
   },
   {
-    "name": "vanuatu",
-    "country_Code": "vu",
-    "alias": ""
+    name: 'vanuatu',
+    country_code: 'vu',
+    alias: '',
   },
   {
-    "name": "vatican.city",
-    "country_Code": "va",
-    "alias": ""
+    name: 'vatican.city',
+    country_code: 'va',
+    alias: '',
   },
   {
-    "name": "venezuela",
-    "country_Code": "ve",
-    "alias": ""
+    name: 'venezuela',
+    country_code: 've',
+    alias: '',
   },
   {
-    "name": "vietnam",
-    "country_Code": "vn",
-    "alias": ""
+    name: 'vietnam',
+    country_code: 'vn',
+    alias: '',
   },
   {
-    "name": "wales",
-    "country_Code": "gb.wls",
-    "alias": ""
+    name: 'wales',
+    country_code: 'gb.wls',
+    alias: '',
   },
   {
-    "name": "wallis.and.futuna",
-    "country_Code": "wf",
-    "alias": ""
+    name: 'wallis.and.futuna',
+    country_code: 'wf',
+    alias: '',
   },
   {
-    "name": "western.sahara",
-    "country_Code": "eh",
-    "alias": ""
+    name: 'western.sahara',
```

Comment:
Replace dots in names with spaces :+1:
I'll later make PR that fixes this in `<Flag>`.